sql/src/main/java/io/crate/plugin/SQLPlugin.java

```diff
@@ -109,6 +109,8 @@ public Settings additionalSettings() {
         settings.add(CrateCircuitBreakerService.QUERY_CIRCUIT_BREAKER_OVERHEAD_SETTING);
         settings.add(DecommissioningService.DECOMMISSION_INTERNAL_SETTING_GROUP);
         settings.add(SQLOperations.NODE_READ_ONLY_SETTING);
+        settings.add(org.elasticsearch.common.settings.Setting.simpleString(MonitorModule.NODE_INFO_EXTENDED_TYPE,
```

Comment:
pls change MonitorModule.NODE_INFO_EXTENDED_TYPE to be a ES setting and use it here